src/plinder/core/utils/schemas.py

```diff
@@ -79,6 +79,8 @@
 
 
 # subject to criteria used in save_linked_structures.py
+# TODO: this schema is now out of date since addition of
```

Comment:
I guess the schema for scores.json should be described in eval instead, since it's the same output as when evaluating predictions